static/panes/tool.js

```diff
@@ -511,6 +512,18 @@ Tool.prototype.onCompileResult = function (id, compiler, result) {
             if (toolResult.sourcechanged && this.editorId) {
                 this.eventHub.emit('newSource', this.editorId, toolResult.newsource);
             }
+            if (toolResult.artifactGenerated) {
+                if (toolResult.artifactGenerated.type === 'application/octet-stream') {
+                    fetch('data:image/jpeg;base64,' + toolResult.artifactGenerated.content)
```

Comment:
Why `image/jpeg` here? That seems disingenuous does `application/binary` not work here (which seems more appropriate?);

Reply:
I did a lot of trial and error and might have stopped too early in the cleaning of the different trials. I'll look into having the actual type here.

Reply:
fixed